app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,8 @@
     <%= csrf_meta_tags %>
     
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+     
+    <script src="https://use.fontawesome.com/49e2c60c0f.js"></script>
```

Comment:
Can you please briefly explain how this works?
I was used to include fontawesome differently.